sycl/include/sycl/feature_test.hpp.in

```diff
@@ -55,7 +55,6 @@ __SYCL_INLINE_VER_NAMESPACE(_V1) {
 #define SYCL_EXT_ONEAPI_SUB_GROUP 1
 #define SYCL_EXT_ONEAPI_PROPERTIES 1
 #define SYCL_EXT_ONEAPI_NATIVE_MATH 1
-#define SYCL_EXT_ONEAPI_BFLOAT16 1
```

Comment:
I think this still needs to be defined, doesn't it?

Reply:
Yes, corrected now.